app/server/Launcher.hs

```diff
@@ -0,0 +1,45 @@
+module Launcher where
+
+import Control.Concurrent.Async
+import Prelude
+import System.Process
+
+-- What is @bracket@?
+--
+-- When you want to acquire a resource, do some work with it, and then release
+-- the resource, it is a good idea to use bracket, because bracket will install
+-- the necessary exception handler to release the resource in the event that an
+-- exception is raised during the computation. If an exception is raised, then
+-- bracket will re-raise the exception (after performing the release).
+--
+-- http://hackage.haskell.org/package/base-4.12.0.0/docs/Control-Exception-Base.html#v:bracket
+
+
+
+
+
+
+
+
+
+-- | Run an external process alongside a haskell IO action
+--
+-- Both are expected to run forever.
+--
+-- Sadly you cannot quit this in ghci
+bracketProcess :: CreateProcess -> IO () -> IO ()
+bracketProcess process action = do
+    _ <- withCreateProcess
+        process {delegate_ctlc = True, std_out = CreatePipe }
```

Comment:
With this `delegate_ctlc`, `ctrl-d` works. `ctrl-c` still doesn't work…

I'd also like to redirect std_out, but don't know how yet.